test/integration/basic_test.go

```diff
@@ -108,11 +120,22 @@ func (m *Minishift) executingCommand(command string) error {
 		cmdErr,
 		cmdExit,
 	}
-
 	// Beware: you are responsible to verify the lastCommandOutput!
 	return nil
 }
 
+func (m *Minishift) executingMinishiftCommandSucceeds(command string) error {
+	err := m.executingCommand(command)
+	if err != nil {
+		return err
+	}
+	if lastCommandOutput.ExitCode != 0 && len(lastCommandOutput.StdErr) != 0 {
```

Comment:
&&? I think it should be || here. If either of the two occurs the test should fail.